queries/latex/textobjects.scm

```diff
@@ -0,0 +1,28 @@
+((environment
+  . (_)
+  . (_) @_start
+  (_) @_end . (_) .
+) @block.outer
+(#make-range! "block.inner" @_start @_end))
+
+((environment
+  (begin
+   name: (word) @_frame)
+  . (_) @_start
+  (_) @_end . (_) .) @frame.outer
```

Comment:
I think you need to update the README with this new group. I have no idea what a frame is in latex or if this group will be useful for other languages

Reply:
I thought this would happen automatically but I need to add it to CONTRIBUTING. The README job now runs only on pushes on master.

I couldn't find an equivalent to it. Maybe there is something similar in RST. If not users can just ignore it.

Reply:
We could also remove it. I thought it would not hurt if we add few filetype specific things users would set mapping manually.

Math would be another thing that would only apply to LaTeX and maybe RST.

Reply:
yeah, it doesn't hurt adding it. I have never used latex